target/product/emulator.mk

```diff
@@ -20,7 +20,22 @@
 
 # Host modules
 PRODUCT_PACKAGES += \
-
+    emulator \
+    emulator-x86 \
+    emulator-arm \
+    emulator-mips \
+    emulator64-x86 \
+    emulator64-arm \
+    emulator64-mips \
+    libOpenglRender \
+    libGLES_CM_translator \
+    libGLES_V2_translator \
+    libEGL_translator \
```

Comment:
I suggest to use alphabetical order

  libEGL_translator \
  libGLES_CM_translator \
  libGLES_V2_translator \
  libOpenglRender \

Here and for lib64 below.